Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -677,6 +677,14 @@ Renderer::Renderer()
       g_Config.backend_info.bSupportsPaletteConversion &&
       g_Config.backend_info.bSupportsComputeShaders && g_ogl_config.bSupportsImageLoadStore;
 
+  // The GPU shader code appears to be context-specific on Mesa/i965.
+  // This means that if we compiled the ubershaders asynchronously, they will be recompiled
+  // on the main thread the first time they are used, causing stutter. Nouveau has been
```

Comment:
Maybe better to put the details on the crash in the DriverDetails entry -- otherwise it will get forgotten over time even when the issue is fixed in newer drivers.